lib/ensemble_app.dart

```diff
@@ -197,3 +197,64 @@ class EnsembleAppState extends State<EnsembleApp> {
         home: Scaffold(backgroundColor: loadingBackgroundColor, body: widget));
   }
 }
+
+class Message {
```

Comment:
Wrong place as these classes are unrelated. Please move them elsewhere to its own class. EnsembleApp is the root widget that folks will integrate with and look at. It should be minimal and make sense.